### PR TITLE
Identify extension appTypes

### PR DESCRIPTION
**Pull request by sean-roberts**
Instrumenting the appType identifier for chrome-extension

### CODE REVIEW

**Comment by chdorner** [comment]
We should probably add `"appType": "firefox-extension"` to the `settings/firefox-*.json` settings.

**Comment by sean-roberts** [comment]
Added

File: settings/chrome-dev.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
 
   "googleAnalytics": "UA-26026798-6",
 
-  "browserIsChrome": true
+  "browserIsChrome": true,
```
Comment (robertknight) [inline settings/chrome-dev.json]: I wondered where `browserIsChrome` is used at first, then I saw that it is used by the extension manifest template. In future we might want to separate out the settings that are only used when building the extension vs. those that are used by the client.

**Comment by robertknight** [comment]
LGTM

**Pull request merged**